zebra/interface.c

```diff
@@ -2861,8 +2861,8 @@ static int if_config_write(struct vty *vty)
 		if (ifp->vrf_id == VRF_DEFAULT)
 			vty_frame(vty, "interface %s\n", ifp->name);
 		else
-			vty_out(vty, "interface %s vrf %s\n", ifp->name,
-				vrf->name);
+			vty_frame(vty, "interface %s vrf %s\n", ifp->name,
```

Comment:
I did this intentionally because `interface X vrf Y` is not a no-op, as it assigns the interface into a VRF